bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -537,9 +551,12 @@ def _handle_image(self, image_msg):
             blue_mask = self._blue_color_detector.get_mask_image()
 
             # Publish mask images
-            self._pub_white_mask_image.publish(self._cv_bridge.cv2_to_imgmsg(white_mask, '8UC1'))
-            self._pub_red_mask_image.publish(self._cv_bridge.cv2_to_imgmsg(red_mask, '8UC1'))
-            self._pub_blue_mask_image.publish(self._cv_bridge.cv2_to_imgmsg(blue_mask, '8UC1'))
+            self._pub_white_mask_image.publish(
+                ros_utils.build_image_msg(image_msg.header, white_mask, '8UC1'))
+            self._pub_red_mask_image.publish(
+                ros_utils.build_image_msg(image_msg.header, red_mask, '8UC1'))
+            self._pub_blue_mask_image.publish(
+                ros_utils.build_image_msg(image_msg.header, blue_mask, '8UC1'))
```

Comment:
please use a separate PR for this stuff in the future.

Reply:
Ok